server/routes/user.js

```diff
@@ -176,7 +176,7 @@ router.patch('/user/:uuid/update/about', (req,res) => {
 })
 
 
-router.get('/user/emailExist', (req,res) => {
+router.post('/user/emailExist', (req,res) => {
```

Comment:
its not a post its a get command and send an email in the param's not in body

Reply:
Done